src/_gettsim/parameters/eink_st.yaml

```
@@ -13,6 +13,8 @@ parameter_einkommensteuertarif:
       §32a EStG
       The quadratic rate is calculated by gettsim with the formula of the
       Progressionsfaktor. For details see the docstring of add_progressionsfaktor.
+  unit: null
+  reference_period: null
```

```suggestion
  reference_period: Year
```

Because ESt formula has yearly zvE thresholds in it